features/organizer/educational-description.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Test updating organizers via complete overwrite
+
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  Scenario: Creating a new educational description
+    Given I create a minimal organizer and save the "url" as "organizerUrl"
+    Given I set the JSON request payload to:
+    """
+    {
+        "educationalDescription": "Nederlandse update"
+    }
+    """
+    When I send a PUT request to "%{organizerUrl}/educational-description/nl"
+    Then the response status should be "204"
+    And I get the organizer at "%{organizerUrl}"
+    Then the JSON response at "educationalDescription" should be:
+    """
+    {
+        "nl": "Nederlandse update"
+    }
+    """
+
+  Scenario: Failing to update educational description with faulty body
+    Given I create a minimal organizer and save the "url" as "organizerUrl"
+    Given I set the JSON request payload to:
+    """
+    {
+        "so-wrong": "Invalid value"
+    }
+    """
+    When I send a PUT request to "%{organizerUrl}/educational-description/nl"
+    Then the response status should be "400"
```

Comment:
We even check the response body. For example https://github.com/cultuurnet/udb3-backend/blob/master/features/place/booking-availability.feature#L17-L25